great_expectations/core/usage_statistics/package_dependencies.py

```diff
@@ -90,7 +90,6 @@ class GXDependencies:
             "openpyxl",
             "pre-commit",
             "psycopg2-binary",
-            "pydantic",  # dev dependency because of mypy plugin
```

Comment:
Is this removed because pydantic is a required dependency?

Reply:
Correct, before our required dependencies weren't being installed as part of the type-checking step, so it was also added here so that we would have the pydantic type info.

Now we do install all the required dependencies in the type-checking step, so this is no longer needed.